Configurations/10-main.conf

```diff
@@ -1345,6 +1345,7 @@ sub vms_info {
             my @ex_libs = ();
             push @ex_libs, 'ws2_32.lib' unless $disabled{sock};
             push @ex_libs, 'gdi32.lib advapi32.lib crypt32.lib user32.lib';
+            push @ex_libs, 'ncrypt.lib';
```

Comment:
> I think it can be argued that crypt32.lib shouldn't be there

I think so, yes. But there is difference. crypt32.lib is present in legacy SDKs, while ncrypt.lib is not. This is kind of the real concern. Having library on command line you don't link with has no side effects (on Windows), but lib-file needs to be present if you refer to in on command line irregardless. In other words it's availability of ncrypt.lib file in user's SDK that is concern.